charts/tractusx-connector-azure-vault/values.yaml

```
@@ -520,7 +520,8 @@ vault:
     transferProxyTokenEncryptionAesKey: transfer-proxy-token-encryption-aes-key
 
 backendService:
-  httpProxyTokenReceiverUrl: ""
+  # Specifies a backend service which will receive the EDR
+  httpProxyTokenReceiverUrl: "httpProxyTokenReceiverUrl"
```

Add `--` so that this will be appear in the README.md in the description column and maybe also add a real url which also indicates that it needs to be changed:

```suggestion
  # -- Specifies a backend service which will receive the EDR
  httpProxyTokenReceiverUrl: "https://example.com"
```